src/postgres-util/src/tunnel.rs

```diff
@@ -229,11 +236,18 @@ impl Config {
                 Ok(client)
             }
             TunnelConfig::AwsPrivatelink { connection_id } => {
-                let (host, port) = self.address()?;
+                // This section of code is somewhat subtle. We are overriding the host
+                // to the PrivateLink host, but leaving the host for TLS verification as
```

Comment:
I feel like there's a few words missing here?

```suggestion
                // This section of code is somewhat subtle. We are overriding the host
                // for the actual TCP connection to be the PrivateLink host, but leaving the host
                // for TLS verification as
```